tensorflow_datasets/image/B_All_White_Blood_Cancer_Microscopic_Image.py

```diff
@@ -0,0 +1,117 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import os
+import tensorflow as tf
+import tensorflow_datasets as tfds
+
+_IMAGE_URL = ("http://people.duke.edu/~yx141/C_NMC_test_prelim_phase_data.zip")
```

Comment:
@us would you know if datasets like this with the right licenses can be hosted on some official tfds bucket?